samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -79,14 +94,37 @@ export class FormComponent implements OnInit {
     this.submittingForm = true;
 
     try {
+      /** This was to be added because when logging in locally by pressing enter it selects the first identity provider by default
```

Comment:
I think we should format the comments in the same style as below.

For example this comment may look like

```
/** *********************************************************************
       * SDK INTEGRATION POINT
       * Summary: Handle selection of IdpCallbacks
       * ----------------------------------------------------------------------
       * Details: If we do not initially select the IdpCallback, when pressing enter,
       * local authentication will automatically be submitted. 
       * This allows us to submit the form with the proper selection of an 
       * identity provider rather than local authentication.                      
 /** *********************************************************************

```
       ********************************************************************* */

Reply:
done